plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -586,6 +586,24 @@ public void testMergeSimpleSelectPartitioned()
         assertUpdate("DROP TABLE " + targetTable);
     }
 
+    @Test
+    public void testTableLocationTrailingSlash()
```

Comment:
The current place is between `testMergeSimpleSelectPartitioned` and `testMergeUpdateWithVariousLayouts`. I would recommend moving to before `testMergeSimpleSelectPartitioned` for grouping MERGE tests.

Reply:
Right, moved